os/board/rtl8721csm/src/component/common/api/wifi/wifi_conf.c

```diff
@@ -78,6 +78,8 @@ extern unsigned char dhcp_mode_sta;
 #endif
 /* The flag to check if wifi init is completed */
 static int _wifi_is_on = 0;
+void* param_indicator;
+struct task_struct wifi_autoreconnect_task;

```

Comment:
If this variable is used only in this file, please add static.

Reply:
Resolved on [daa8fd5fd2cb81556c031b48f6eb3222ba653f39]